test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -221,6 +225,10 @@ void doBlockchainTestNoLog(json_spirit::mValue& _v, bool _fillin)
 		}
 	}
 
+	//Delete source test from the json
+	for (auto i: erase_list)
+		_v.get_obj().erase(i);
```

Comment:
Not in a map, I read https://stackoverflow.com/a/263958/103518.  Maybe I need to check better references.